upgrade_through_versions_test.py

```diff
@@ -188,7 +54,7 @@ def data_writer(tester, to_verify_queue, verification_done_queue, rewrite_probab
     Intended to be run using multiprocessing.
     """
     # 'tester' is a cloned object so we shouldn't be inappropriately sharing anything with another process
-    session = tester.patient_cql_connection(tester.node1, keyspace="upgrade", protocol_version=PROTOCOL_VERSION)
+    session = tester.patient_cql_connection(tester.node1, keyspace="upgrade", protocol_version=tester.protocol_version)
```

Comment:
I think `data_writer` should be rewritten to take a `node` and (optionally, maybe?) a `protocol_version` -- no need to couple this to the `Tester` implementation.

Actually, there's no reason for `patient_cql_connection` to be coupled to `Tester`:

https://github.com/riptano/cassandra-dtest/blob/master/dtest.py#L416

So, I think the thing to do is, somewhere, assign `patient_cql_connection` to a standalone function name, then use that here. That way we get to not care about the `tester` at all, which seems better if we want to not care about how this is used across processes.